llvm/test/CodeGen/SPIRV/hlsl-intrinsics/round.ll

```diff
@@ -0,0 +1,20 @@
+; RUN: llc -O0 -mtriple=spirv-unknown-linux %s -o - | FileCheck %s
+
+; CHECK: OpExtInstImport "GLSL.std.450"
+
+define noundef float @round_float(float noundef %a) {
+entry:
+; CHECK: %[[#]] = OpExtInst %[[#]] %[[#]] Round %[[#]]
```

Comment:
While this test passes, I'm not sure if this is correct based on this issue: https://github.com/microsoft/DirectXShaderCompiler/issues/4368

And shade playground is also reporting RoundEven: https://shader-playground.timjones.io/3e6db4d9a4c4dfe4974c203f721a8c73

Reply:
Should HLSL `round` be lowered to `llvm.roundeven`?

https://learn.microsoft.com/en-us/windows/win32/direct3dhlsl/dx-graphics-hlsl-round
https://llvm.org/docs/LangRef.html#llvm-roundeven-intrinsic

Reply:
we are using `__builtin_elementwise_round` which maps to `intrinsics::round` We would need to use `__builtin_elementwise_roundeven` in  `clang/lib/Headers/hlsl/hlsl_intrinsics.h` but yeah that would work. I can file an issue for this.

Reply:
I'll fix up this test case to use the right intrinsic in the next revision.